staging_service/autodetect/Mappings.py

```diff
@@ -7,6 +7,9 @@
 EXCEL = "EXCEL"
 ZIP = "CompressedFileFormatArchive"
 
+# Data Transfer Service Manifest format (which is a specific JSON format)
+DTS_MANIFEST = "JSON"
+
```

Comment:
These types are just the file types, regardless of the semantics of the type. Just JSON should be used here

Reply:
Done.